packages/pyrtl/test_pyrtl.py

```diff
@@ -0,0 +1,39 @@
+from pyodide_build.testing import run_in_pyodide
+
+@run_in_pyodide(packages = ["pyrtl"])
+def test_pyrtl():
+    import pyrtl
+
+    # Calculate GCD of a and b
+    a, b, begin = pyrtl.input_list('a/8 b/8 begin/1')
+    gcd, done = pyrtl.output_list('gcd/8 done/1')
+
+    x = pyrtl.Register(len(a))
+    y = pyrtl.Register(len(b))
+
+    with pyrtl.conditional_assignment:
+        with begin:
+            x.next |= a
+            y.next |= b
+        with x > y:
+            x.next |= x - y
+        with y > x:
+            y.next |= y - x
+        with pyrtl.otherwise:
+            done |= True
+
+    gcd <<= x
```

Comment:
That would be a syntax error unless I'm missing something?

Reply:
This isn't a syntax error. At that point, `done` and `gcd` are WireVectors, which have `__ilshift__` and `__ior__` (. (among many other methods) defined to do the behind-the-scenes magic of creating netlists behind the scenes